packages/back-end/src/controllers/organizations.ts

```diff
@@ -217,14 +219,15 @@ export async function getActivityFeed(req: AuthRequest, res: Response) {
   const { org, userId } = getOrgFromReq(req);
   try {
     const docs = await getWatchedAudits(userId, org.id, {
-      limit: 50,
+      limit: 25,
```

Comment:
I didn't think about limits before when suggesting multiple queries.  That could get into a weird situation where a company has only used experiments recently, so their activity feed shows 25 experiment events and then a huge time gap and then super old feature events.

I'm thinking we can change the activity page to use date instead of a limit.  So it can show all events in the last 7 days for example.  I think that makes sense for the page anyway and avoids the problem above.